app/routes/dev.tsx

```diff
@@ -12,14 +12,13 @@ import logger from '~/lib/logger.server';
 import { requireUser, requireUsername } from '~/session.server';
```

Comment:
I'm not sure about others, but I'm ok to remove this dev page.  @SerpentBytes @dadolhay

Reply:
I'll leave it in here for now, but we can rip it out later this week or next.